react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -1206,7 +1208,7 @@ const ResourceAllocationFormItems: React.FC<
         </Card>
       ) : null}
       {/* TODO: Support cluster mode */}
-      {!baiClient._config.hideAgents && (
+      {enableAgentSelect && (
         <Form.Item
           label={t('session.launcher.SelectAgent')}
           required
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +1220]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This is a screenshot when baiClient.supports('agent-select') is false and hideAgent=false.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/4a27d183-e293-449e-a71a-b88949bcc2ce.png)

Please consider this case.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2870?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->